map-utils.js

```diff
@@ -1,126 +1,140 @@
-var lng = (point.x - tilePixelCenter) / pixelsPerLngDegree;
-var latRadians = (point.y - tilePixelCenter) / -pixelsPerLngRadian; 
-var lat = radiansToDegrees(2 * Math.atan(Math.exp(latRadians)) - Math.PI / 2);
-return {lat: lat, lng: lng} 
-}
+  var lng = (point.x - tilePixelCenter) / pixelsPerLngDegree;
+  var latRadians = (point.y - tilePixelCenter) / -pixelsPerLngRadian;
+  var lat = radiansToDegrees(2 * Math.atan(Math.exp(latRadians)) - Math.PI / 2);
+
+  return { lat: lat, lng: lng };
+};
 var getFormattedPlaceName = function (placeObj) {
-if (!placeObj.formatted_address) {
-return null
-}
-var htmlAddress = placeObj.adr_address
-var firstPart = htmlAddress.substr(0, htmlAddress.indexOf("<"))
-var addressContainer = window.document.createElement("span")
-addressContainer.innerHTML = htmlAddress
-var childNodes = Array.prototype.slice.call(addressContainer.children, 0)
-if (childNodes.length == 0) {
-return placeObj.formatted_address
-}
-return firstPart + childNodes
-  .filter(function (node) { return node.className != "postal-code"})
-  .map(function (node) { return node.textContent})
-  .join(", ")  
-}
+  if (!placeObj.formatted_address) {
+    return null;
+  };
+  var htmlAddress = placeObj.adr_address;
+  var firstPart = htmlAddress.substr(0, htmlAddress.indexOf('<'));
+  var addressContainer = window.document.createElement('span');
+  addressContainer.innerHTML = htmlAddress;
+  var childNodes = Array.prototype.slice.call(addressContainer.children, 0);
+  if (childNodes.length === 0) {
+    return placeObj.formatted_address;
+  };
+
+  return firstPart + childNodes
+    .filter(function (node) { return node.className !== 'postal-code';})
+    .map(function (node) { return node.textContent;})
+    .join(', ');
+};
 var getAdjustedMapCenter = function (options) {
-var mapCenter = options.mapCenter;
-var mapZoom = options.mapZoom;
-var offset = options.offset;
-var reverse = options.reverse;
-var directionalOffset = reverse ? offset * -1 : offset
-z = Math.pow(2, mapZoom)
-console.log('z: ', z);
-var point = getPointFromLatLng({
-lat: result(mapCenter, "lat"),
-lng: result(mapCenter, "lng"),
-}, {asPlainObject: true})
-var adjustedMapCenter = getLatLngFromPoint({
-x: point.x,
-y: point.y + (directionalOffset / 2) / z,
-})
-adjustedMapCenter.lat = round(adjustedMapCenter.lat, 6)
-adjustedMapCenter.lng = round(adjustedMapCenter.lng, 6)
-return adjustedMapCenter      
-}
+  var mapCenter = options.mapCenter;
+  var mapZoom = options.mapZoom;
+  var offset = options.offset;
+  var reverse = options.reverse;
+  var directionalOffset = reverse ? offset * -1 : offset;
+  var z = Math.pow(2, mapZoom);
+  //console.log('z: ', z);
+  var point = getPointFromLatLng({
+    lat: mapCenter.lat,
+    lng: mapCenter.lng
+  }, { asPlainObject: true });
+  var adjustedMapCenter = getLatLngFromPoint({
+    x: point.x,
+    y: point.y + (directionalOffset / 2) / z
+  });
+  adjustedMapCenter.lat = Math.round(adjustedMapCenter.lat, 6);
```

Comment:
This is an interesting approach. Essentially the same comment applies as with `round()` in that this was originally imported from lodash through a require statement. Since there's no way you could know this, the best approach would be to assume it was declared globally (which technically it was.)